Lombiq.Npm.Targets.targets

```diff
@@ -36,11 +36,11 @@
   <Target
     Name="NpmDotnetPrebuild"
     AfterTargets="InitCommonNodeModules;NpmInstall"
-    BeforeTargets="EmbeddModuleAssets;BeforeBuild"
+    BeforeTargets="EmbeddModuleAssets;Compile"
     Inputs="@(NpmDotnetPrebuildWatchedFiles)"
     Outputs="$(NpmDotnetPrebuildStampFile)"
     Condition="'$(ExecDotnetPrebuildCommand)' == 'true' AND Exists('package.json')">
-    <Exec Command="$(NpmDotnetPrebuildCommand)" ConsoleToMSBuild="true">
+    <Exec Command="$(NpmDotnetPrebuildCommand)" ConsoleToMSBuild="true" ContinueOnError="true">
```

Comment:
It's not due to that but due to the error in the stylelint CLI (for example) which will break the build. Converting it to a warning should be fine for local development. The warning will be turned into an error again in CI.

Reply:
Do you mean linter violations or other errors?

Reply:
Yes, linter violations. But I suppose any of the used cli tools that exit with a non-zero exit code will trigger the same behavior.

Reply:
My problem with this is that we're not distinguishing between showstopper errors and things like linter violations that while should be shown, don't actually stop anything. Since this is a generic module for any NPM (Node) process, we shouldn't generally mask all errors: That would lead to something properly breaking while the build continue and you don't see it until you look at the warnings.

Reply:
🤔

Reply:
I understand the problem. I'll look into it.

Reply:
Thanks!